Mathlib/FieldTheory/SeparableDegree.lean

```diff
@@ -18,78 +18,113 @@ This file contains basics about the separable degree of a field extension.
 
 ## Main definitions
 
-- `Field.Emb F E`: the type of `F`-algebra homomorphisms from `E` to the algebraic closure of `E`.
+- `Field.Emb F E`: the type of `F`-algebra homomorphisms from `E` to the algebraic closure of `E`
+  (mathematically, it should be the algebraic closure of `F`, but in order to make its cardinality
+  has the same type with `Module.rank F E`, we use the algebraic closure of `E`).
```

Comment:
see https://github.com/leanprover-community/mathlib4/pull/8117#discussion_r1381535021 :

> > Regarding
> > > Mathematically, it should be the algebraic closure of F, but in order to make the type compatible with Module.rank F E, we use the algebraic closure of E.
> > 
> > 
> > I don't quite understand the difficulty of working with `AlgebraicClosure F`. I think it would make the inductive step to show multiplicativity of separable degree slightly easier.
> 
> Because the type of `Cardinal.mk (E →ₐ[F] (AlgebraicClosure F))` is `Cardinal.{max u v}`, while the type of `Cardinal.mk (E →ₐ[F] (AlgebraicClosure E))` is `Cardinal.{v}`, which is the same as `Module.rank F E`. This allows us to write `sepDegree F E = Module.rank F E` without mentioning `Cardinal.lift`.

Oh but that was a historical remark, since now `sepDegree` gets a new definition.